### DISCUSSION TITLE
vercel github bot comments wrong preview link.

### DISCUSSION THREAD

**Discussion by disjukr** [vercel/community#3154] [Bug Report]
### Page to Investigate

https://github.com/portone-io/developers.portone.io/pull/90#issuecomment-1624585747

### Steps to Reproduce

It seems that the problem started today. The Vercel bot started putting the same link in `Add feedback` into `Visit Preview`.
Since we are a public project, anyone should be able to access this preview link, but the link dedicated to `Add feedback` requires Vercel authentication, which puts us in trouble.

**Comment by amyegan** [comment]
Thanks for reporting this 🙇

**Reply by amyegan** [reply]
The team found the service that was causing this problem and fixed it. The login page should no longer appear on the preview link for future commits. Please let me know if you still have trouble with new commits.
